Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/BoundsHandleVisual.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+{
+    /// <summary>
+    /// Class identifying common handles behavior
+    /// </summary>
+    public class BoundsHandleVisual : MonoBehaviour
```

Comment:
Some thoughts on naming - this is named very specifically to the object that it applies to, rather than the behavior that it triggers. The thing to think about here is if we ever believe that we'll want this type of behavior on other things going forward (i.e. if we will want other entities that ignore near interaction to some way), then we'll either have to:

1. Create a new script that is also an empty monobehavior with a new name
2. Refactor in the way above.
3. Use BoundsHandleVisual on the new thing and be a bit confused.

In general if we think about capabilities/parameters/things it's better to name them based on the behaviors that they trigger rather than the scenarios that they are part of. Scenarios can change. Behaviors are generally more persistent and durable/reusable across scenarios

Reply:
So more concretely, maybe something like "IgnoresNearGrabInteraction" or something with a better name I'm actually bad at the words part

Reply:
Or maybe IgnoresSphereQuery or i dunno.